docs/admin/authentication.md

```diff
@@ -40,6 +40,29 @@ option to apiserver. The referenced file must contain one or more certificates a
 to use to validate client certificates presented to the apiserver. If a client certificate
 is presented and verified, the common name of the subject is used as the user name for the
 request.
+When using client certificate authentication, you should generate certificates by deployment
+script or yourself. The deployment script is implemented at
+`cluster/saltbase/salt/generate-cert/make-ca-cert.sh`
+Execute this script with two parameters. First is the `cert_ip` , second is the
+`subject-alt-name`. `cert_ip` used to generate the CA default CN and append to
+`subject-alt-name` for subjectAltName . http://wiki.cacert.org/FAQ/subjectAltName
```

Comment:
make subjectAltName link to the wikipedia article by saying

[subjectAltName](http://...).